jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/subpackage/ServerDefaultResourcesTest.java

```diff
@@ -0,0 +1,64 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.server.subpackage;
+
+import java.util.stream.Stream;
+
+import org.eclipse.jetty.server.Server;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class ServerDefaultResourcesTest
+{
+    public static Stream<Arguments> arguments()
+    {
+        return Stream.of(
+            new Server(),
+            new Server(){}
+        ).map(Arguments::of);
```

Comment:
I don't think this tests what you think it tests.

Reply:
@joakime this test fails before fix and now passes.

The idea is that it is trying to load `jetty-dir.css`, and it does `getClass().getResource(name)`, internally this appends the package name as the resource does not start with `/`, and because the server is subclassed with the `{}` it is looking for `/org/eclipse/jetty/server/subpackage/jetty-dir.css` which does not exist.

Reply:
So the second argument is an anonymous subclass. got it.